src/client/client.cpp

```diff
@@ -561,6 +568,66 @@ EngineStatus Client::currentStatus() const
     return m_status;
 }
 
+
+// uses fast voxel traversal to get the voxels intersected by a ray of length `range`
+// assumes blocks are 1x1x1
+std::vector<BlockPosition> Client::getIntersectedBlocks(float range) {
+    auto startPoint = mp_player->position;
+    auto direction  = glm::normalize(forwardsVector(mp_player->rotation));
+    auto endPoint   = startPoint + direction * range;
+    auto startVoxel = toBlockPosition(startPoint);
```

Comment:
Just thinking a tiny change here.

I feel this function can be moved to be not a member of the Client class, but instead become a "free" function that takes in a position and direction as a parameter.

Reply:
Would it be best to add this function to `maths.cpp` and 'maths.h`?

Reply:
Yeahh that would a good place for it, thanks :)